src/components/targets-map/MarkerTargets.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { CircleMarker, Marker } from 'react-leaflet';
+
+import {
+  safron,
+  createTargetOpacity,
+} from '../../style/common/_constants.scss';
+import { getIconByTopic } from './map-icons';
+
+const MarkerTarget = ({ targets }) => (
+  <div>
+    {targets.map(({ _, topic, radius, lat, lon }) => (
+      <div>
```

Comment:
missing key attribute

Reply:
What do you mean?

Reply:
https://en.reactjs.org/docs/lists-and-keys.html take a look at _keys_ section. As for prop types I mean add MarkerTarget.propTypes = { targets: array }. Do you remember that? I guess we discussed about it

Reply:
Oh thanks! Yes I'll add the propTypes.
About the keys, is it wrong to use `map` in this way: `targets.map(({ _, topic, radius, lat, lon })`?